compiler/rustc_typeck/src/check/upvar.rs

```diff
@@ -613,11 +614,6 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
     /// using list of `Projection` slices), it returns true if there is a path that is not
     /// captured starting at this root variable that implements Drop.
     ///
-    /// FIXME(project-rfc-2229#35): This should return true only for significant drops.
```

Comment:
Don't remove the actual text, turn this into a statement like "the function only returns true for significant drops, A drop ...."